moshi/moshi/models/tts.py

```diff
@@ -676,3 +685,124 @@ def get_prefix(self, audio_path: Path) -> torch.Tensor:
         null_text = torch.full_like(prefix[:1], self.machine.token_ids.zero)
         prefix = torch.cat([null_text, prefix], dim=0)
         return prefix
+
+    def simple_generate(
+        self, text: str | list[str], voice: str | list[str], cfg_coef: float = 2.0, show_progress: bool = True
+    ) -> list[torch.Tensor]:
+        """Generate audio directly from text and voice name(s).
+
+        `text` and `voice` can be "broadcast" against each other like in NumPy: if one
+        is a list and the other a single item, the single item is repeated for each item
+        in the list. If both are lists, they must have the same length and voice[i]
+        will be used for text[i].
+
+        If you need more control, refer to the examples in
+        https://github.com/kyutai-labs/delayed-streams-modeling
+
+        Args:
+            text: text or list of texts to synthesize.
+            voice: voice name or list of voice names to use. Each entry can be a path
```

Comment:
actually for the public model, voice could be a path to a wav file!
Also if someone wants to use a local voice, they can use `'file://' + path` as a voice... It does get a bit complex ^^'